hapi/hedera-protobufs/platform/event/tss_message_transaction.proto

```diff
@@ -10,7 +10,7 @@
  */
 syntax = "proto3";
 
-package proto;
+package com.hedera.hapi.platform.event;
```

Comment:
will a tss message be a system transaction? if yes, this makes sense. if no, i think it should be part of another package

Reply:
Yes I believe this will be a system transaction

Reply:
@edward-swirldslabs can you confirm?

Reply:
@mhess-swl, @lpetrovic05, @rbair23, @tinker-michaelj, @jsync-swirlds, @Neeharika-Sompalli, @thomas-swirlds-labs, @anthony-swirldslabs, @kfa-aguda, 

While I called them `System Transactions` in my proposal, and I am strongly of the opinion that they are NOT supposed to be `User Transactions`, the only `System Transactions` in the code are platform transactions handled by the platform, and after the redesign to make TSS a Service, the `TssMessageTransaction` and the `TssVoteTransaction` are now [user transactions](https://github.com/hashgraph/hedera-services/blob/273d6e8d20d3a48ad67feb55e5b65d63f57490c6/hapi/hedera-protobufs/services/transaction_body.proto#L425-L433).  They are the only user transactions that are [restricted to "0-0"](https://github.com/hashgraph/hedera-services/blob/273d6e8d20d3a48ad67feb55e5b65d63f57490c6/hedera-node/hedera-config/src/main/java/com/hedera/node/config/data/ApiPermissionConfig.java#L265-L266), which I assume means that no user has permission to submit. 

We do not have the equivalent of "OS level" transactions in the application space, otherwise I would say they go there.  

At minimum, the `ShareSignatureTransaction` (future TSS development) will need to be submitted to the transaction pool with highest priority and go to the front of the queue of waiting transactions to get into the next event. 

I can imagine that there will be more transactions in the future that are "OS level", internal system or OS operation.  

But as of right now `SystemTransaction` means platform-only created and handled transactions.  

The proposal should probably drop the use of  `SystemTransaction` as a term, but there is no other term to replace it with to convey how much this is NOT a user transaction...

Reply:
Resolved by moving these transactions to the `auxiliary` folder